Trappist/src/Promact.Trappist.Core/Controllers/CategoryController.cs

```diff
@@ -91,12 +91,17 @@ public CategoryController(ICategoryRepository categoryRepository, IStringConstan
                 return BadRequest(ModelState);
             }
             Category category = await _categoryRepository.GetCategoryByIdAsync(categoryId);
```

Comment:
IsCategoryExistAsync(string categoryName, int id) method has a two parameters categoryName and id.
in delete process server catch only id for remove category.